src/modules/infobox/components/FeatureList.vue

```diff
@@ -25,6 +25,15 @@ function generateFeatureIdForList(feature, indexInList) {
     }
     return featureId
 }
+function highlightFeature(feature) {
+    store.dispatch('setHighlightedFeatureId', {
+        featureId: feature?.id,
+        dispatcher: 'FeatureList.vue',
+    })
+}
+function clearHighlightedFeature() {
+    store.dispatch('setHighlightedFeatureId', { featureId: null, dispatcher: 'FeatureList.vue' })
+}
```

Comment:
Based on your comment of #644 I would use `highlightedFeatureId` instead of `featureId`